libvmi/driver/kvm/kvm.c

```diff
@@ -1364,6 +1380,17 @@ kvm_init_vmi(
     }
     vmi->num_vcpus = info.nrVirtCpu;
 
+#ifndef HAVE_LIBVMI_REQUEST
+    char *version = exec_info_version(kvm);
+    dbprint(VMI_DEBUG_KVM, "--Checking QEMU version string : %s", version);
+    // find substring "vmi"
```

Comment:
Ok, so i will remove `vmi` to the QEMU version string.
And to check that the QEMU version is `2.8.0` or above, we need to parse it.

As i would like to avoid manually parsing a string in C, because it's error prone and painful, i would rather use the QMP command `query-version` and load the json output:
~~~JSON
{"return":{"qemu":{"micro":0,"minor":8,"major":2},"package":""},"id":"libvirt-35"}
~~~
which will allow me to have the `major`, `minor` and `micro` int version number directly.
This would add a dependency on a JSON library.

Do we already use a JSON parser in libvmi ?
What do you think of this solution ?